README.md

```diff
@@ -23,6 +23,14 @@ Run this command to install the CLI:
 
     curl -sS https://platform.sh/cli/installer | php
 
+### Installing via multiple PHP commands
+
+You may instead need these commands in some shells (such as MINGW64 on Windows):
+
+    php -r "copy('https://platform.sh/cli/installer', 'platformsh-cli-installer.php');";
```

Comment:
Unfortunately Git Bash's mintty/mingw64 prevents copying and pasting more than 1 line of code at a time